services/wallet/bridge/cbridge.go

```diff
@@ -60,43 +63,27 @@ func (s *CBridge) Name() string {
 }
 
 func (s *CBridge) estimateAmt(from, to *params.Network, amountIn *big.Int, symbol string) (*cbridge.EstimateAmtResponse, error) {
-	client := &http.Client{
-		Timeout: time.Second * 5,
-	}
 	base := baseURL
 	if from.IsTest {
 		base = testBaseURL
 	}
-	url := fmt.Sprintf(
-		"%s/v2/estimateAmt?src_chain_id=%d&dst_chain_id=%d&token_symbol=%s&amt=%s&usr_addr=0xaa47c83316edc05cf9ff7136296b026c5de7eccd&slippage_tolerance=500",
-		base,
-		from.ChainID,
-		to.ChainID,
-		symbol,
-		amountIn.String(),
-	)
 
-	req, err := http.NewRequest(http.MethodGet, url, nil)
-	if err != nil {
-		return nil, err
-	}
+	params := url.Values{}
+	params.Add("src_chain_id", strconv.Itoa(int(from.ChainID)))
```

Comment:
same for these, can't we just add full name params? :)
Why are some using this approach and some not? consistency adds a lot of value imo

Reply:
`src_chain_id ` this is not something that we defined, this is the param requested by clear API and it must be like that.

Reply:
Yep but the many cases below this too